prometheus-exporters/ucs-exporter/alerts/service.alerts

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: license.alerts
+  rules:
+  - alert: UcsServerPortLicenseExpired
+    expr: >
+      ucs_server_license{port_name=~".+"} == 0
+    for: 5m
+    labels:
+      severity: info
+      context: "ucs-exporter"
+      meta: "ucs server {{ $labels.server }} port {{ $labels.port_name }} license is expired."
+    annotations:
+      description: "ucs server {{ $labels.server }} port {{ $labels.port_name }} license is expired."
+      summary: "ucs server {{ $labels.server }} port {{ $labels.port_name }} license is expired."
```

Comment:
Can you add a hint what needs to be done or is not working? I don't  see a playbook linked yet. Something like:
`license is expired, server ran out of support` (I am making something up here)

Reply:
I checked with Kapil and got to know that, It doesnt affect the vpod traffic, but to be compliant we need to add the new license. I have edited the message, pls check.